modules/collection/programs/beets.nix

```
@@ -0,0 +1,75 @@
+{
+  lib,
+  config,
+  pkgs,
+  ...
+}: let
+  inherit (lib.modules) mkIf;
+  inherit (lib.options) mkOption mkEnableOption mkPackageOption literalExample;
+
+  yaml = pkgs.formats.yaml {};
+
+  cfg = config.rum.programs.beets;
+in {
+  options.rum.programs.beets = {
+    enable = mkEnableOption "beets";
+
+    package = mkPackageOption pkgs "beets" {
+      extraDescription = ''
+        To get plugins to work, you will need to override the beets derivation
+        with the plugins you want. Consult the [beets derivation] for a list of
+        available plugins.
+
+        [beets derivation]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/audio/beets/builtin-plugins.nix
+      '';
+      example = literalExample ''
```

mkPackageOption already renders the example a literalExample.
```suggestion
      example = ''
```

---

Thanks for pointing that out, did not know about this behaviour, which makes sense in hindishgt, thought nezia omitted it to avoid typing more